syntax/attribute.tex

```diff
@@ -42,6 +42,14 @@ \subsection{Depth-First Visit Conditions (one-sweep)}
 
 There is an arc $D_i \rarr D_j$ in $sibl_p$ iff in $dep_p$ there is an arc $\alpha_i \rarr \beta_j$ with $\alpha_i \in \text{attr}(D_i)$ and $\beta_j \in \text{attr}(D_j)$.
 
+\subsection{One-Sweep condition}
+
+\begin{itemize}
+  \item NO circular dependences
+  \item Right attributes depend only on right ones in the parent node
+  \item Left attributes depend only on attributes (both right and left) in the siblings
+\end{itemize}
+
```

Comment:
This is basically the same thing as the paragraph immediately above. But I see that this way it's easier to read.
Maybe you can join these 2 subsections (if it's better looking, otherwise it doesn't really matter).

In any case make sure the fifth page does not cut a paragraph in half (like it does now), you can forse a page break with `\newpage`.